runtime/moonbase/src/precompiles.rs

```diff
@@ -108,6 +109,11 @@ type MoonbasePrecompilesAt<R> = (
 	PrecompileAt<AddressU64<1024>, Sha3FIPS256, (CallableByContract, CallableByPrecompile)>,
 	RemovedPrecompileAt<AddressU64<1025>>, // Dispatch<R>
 	PrecompileAt<AddressU64<1026>, ECRecoverPublicKey, (CallableByContract, CallableByPrecompile)>,
+	PrecompileAt<
+		AddressU64<1027>,
+		StorageCleanerPrecompile<R>,
+		(CallableByContract, CallableByPrecompile),
```

Comment:
```suggestion
		 CallableByPrecompile,
```

We don't need to allow SC to call this precompile, for security it's better to not allow what we don't strictly need

Reply:
Change made.